### ISSUE TITLE
glibc: error nixops deploy

### ISSUE THREAD

**Issue by Izorkin** [NixOS/nixpkgs#79849]
Error deploy virtual machines with nixops:
```
vm01-web.> error: Traceback (most recent call last):
  File "/nix/store/q8mf0lbfmivwhi81aj8pxc6xr9130jvp-nixops-1.7/lib/python2.7/site-packages/nixops/deployment.py", line 743, in worker
    raise Exception("unable to activate new configuration (exit code {})".format(res))
Exception: unable to activate new configuration (exit code 1)
```
nix-shell -p nix-info --run "nix-info -m"
```
these paths will be fetched (0.05 MiB download, 0.28 MiB unpacked):
  /nix/store/bgpkj0876waccxc1vrxj76b9kvvv21hd-bash-interactive-4.4-p23-dev
copying path '/nix/store/bgpkj0876waccxc1vrxj76b9kvvv21hd-bash-interactive-4.4-p23-dev' from 'https://cache.nixos.org'...
bash: symbol lookup error: /nix/store/y9zg6ryffgc5c9y67fcmfdkyyiivjzpj-glibc-2.27/lib/libpthread.so.0: undefined symbol: __libc_vfork, version GLIBC_PRIVATE
```
Nix-channels - https://nixos.org/channels/nixos-unstable-small

**Comment by vcunat** [comment]
Well, nixos-unstable now updated glibc version #66528.  Still, I think this would have to be some kind of impurity unknown to me so far.

**Comment by ajs124** [comment]
I saw this exact error message `bash: symbol lookup error: /nix/store/[…]/lib/libpthread.so.0: undefined symbol: __libc_vfork, version GLIBC_PRIVATE` with an `LD_PRELOAD`ed libjemalloc.

In my case, I just disabled that. Idk how or why this might be related to what you're experiencing, but considering the internet didn't seem to know anything about this error message, I guess they're related somehow, because that was also a few days ago with something from nixos-unstable-small.
